Yoast/wordpress-seo
# Created a composer-artifact grunt command and removed the distribution files

## Summary

This PR can be summarized in the following changelog entry:

* Created the composer-artifact grunt command
* Added the distribution files to the gitignore

## Relevant technical choices:

* Making a new entry in the grunt-copy configuration

## Test instructions

This PR can be tested by following these steps:

1. Running `composer install`, `yarn` and `grunt clean:artifact`
2. Running the new `grunt copy:composer_artifact` command.
Ensure the right files are copied to the artifact folder.
3. Running the new `grunt composer-artifact` alias command containing the `copy:composer_artifact` command.
Ensure there is a zip file created with the right files
## Quality assurance


Fixes #9119
